### PR TITLE
Gp2 3808 focus issue

### PR DESCRIPTION
**Pull request by seijihg**
Safari outline fix

Before:
Screenshot 2021-11-18 at 13 33 22

After:
Screenshot 2021-11-18 at 13 32 57

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/GP2-3808
- [x] Jira ticket has the correct status.
- [x] [Changelog](CHANGELOG.md) entry added.

### Reviewing help

- [ ] Explains how to test locally, including how to set up appropriate data
- [x] Includes screenshot(s) - ideally before and after, but at least after

### Housekeeping

- [ ] Added all new environment variables to Vault.
- [ ] Cleaned up old feature flags
- [ ] Upgraded any vulnerable dependencies.
- [ ] Ran the `make manage download_geolocation_data` command
- [ ] I have updated security dependencies
- [ ] Python requirements have been re-compiled.
- [x] Frontend assets have been re-compiled.

### Merging

- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

File: react-components/src/components/FormElements/index.jsx [hunk 1]
```diff
@@ -50,6 +50,7 @@ export const FormElements = memo(
           }
           return (
             <Component
+              className="form__border"
```
Comment (bobmeredith) [inline react-components/src/components/FormElements/index.jsx:53]: should this be `form--border` to hit the selector above?

File: core/sass/components/_forms.scss [hunk 2]
```diff
@@ -45,6 +45,13 @@
       @extend .m-t-0;
     }
   }
+
+  &__border {
+    border-color: #8f89a4;
```
Comment (bobmeredith) [inline core/sass/components/_forms.scss:50]: Looks a bit odd having a hex colour in here.  Is it necessary?
I thought the issue is the outline.
If it's necessary to hide the border, can't you just set to none, or colour to transparent

File: react-components/src/components/FormElements/index.jsx (hunk 1 above)
Comment (seijihg) [reply react-components/src/components/FormElements/index.jsx:53]: this stuffs are from contrast ticket not sure why it got mixed up to this one lol

**Review by bobmeredith — APPROVED** [review]
LGTM

**Pull request merged**